nats-base-client/nats.ts

```diff
@@ -504,4 +508,34 @@ export class NatsConnectionImpl implements NatsConnection {
   get features(): Features {
     return this.protocol.features;
   }
+
+  get services(): ServicesAPI {
+    if (!this._services) {
+      this._services = new ServicesFactory(this);
+    }
+    return this._services;
+  }
+}
+
+export class ServicesFactory implements ServicesAPI {
+  nc: NatsConnection;
+  constructor(nc: NatsConnection) {
+    this.nc = nc;
+    console.log(
+      `\u001B[33m >> service framework is beta functionality \u001B[0m`,
```

Comment:
a nit, but maybe `experimental functionality` would be a better term?
I usually assume that `beta` means complete and more or less stable API, but requires testing and hardening :).

Reply:
I had that before - I think we are calling it beta

Reply:
ok.